tests/common/snappi_tests/qos_fixtures.py

```diff
@@ -25,17 +25,17 @@ def prio_dscp_map(duthosts, rand_one_dut_hostname):
         Example: {0: [0], 1: [1], 2: [2], 3: [3], 4: [4] ....}
     """
     duthost = duthosts[rand_one_dut_hostname]
-    config_facts = duthost.config_facts(host=duthost.hostname, asic_index=0,
+    config_facts = duthost.config_facts(host=duthost.hostname,
                                         source="running")['ansible_facts']
 
-    if "DSCP_TO_TC_MAP" not in list(config_facts.keys()):
+    if "DSCP_TO_TC_MAP" not in config_facts.keys():
```

Comment:
why are we reverting back to just keys()

Reply:
fixed this